payments/management/command_util.py

```diff
@@ -0,0 +1,56 @@
+import logging
```

Comment:
Should this file be something like`base_expiration_command.py`?

Reply:
fixed